test/oas3/execute/main.js

```diff
@@ -524,6 +524,97 @@ describe('buildRequest - OpenAPI Specification 3.0', () => {
         method: 'GET',
       });
     });
+
+    it('should encode JSON values provided as objects', () => {
```

Comment:
Can we create this test just on top of https://github.com/swagger-api/swagger-js/blob/e943f35aa94745f9b7e89bec5a029a89e1789951/test/oas3/execute/main.js#L666 like this?

```js
  describe('`schema` parameters', () => {
    it('should serialize JSON values provided as objects', () => {
....
```